io.github.Faithlife_Community.OuDedetai.yml

```diff
@@ -0,0 +1,106 @@
+      - mkdir -p /app/lib/dri/intel-vaapi-driver
+      - mkdir -p /app/lib/i386-linux-gnu/dri/intel-vaapi-driver
+      - mkdir -p /app/lib/udev/rules.d
+      - mkdir -p /app/extensions/Plugins
+      - mkdir -p /app/lib64
+      - mkdir -p /app/lib32
+      - mkdir -p /app/lib/ffmpeg
+      - mkdir -p /app/lib32/ffmpeg
+      - mkdir -p /app/utils
+      - install -Dm644 ld.so.conf ${FLATPAK_DEST}/etc/ld.so.conf
+      - mkdir -p ${FLATPAK_DEST}/{,lib/debug/}lib/i386-linux-gnu/GL
+      - mkdir -p ${FLATPAK_DEST}/dlls
+    sources:
+      - type: inline
+        dest-filename: ld.so.conf
+        contents: |
+          /app/lib32
+          /app/lib/i386-linux-gnu
+  - name: oudedetai
+    buildsystem: simple
+    build-commands:
+      # Our installer expects sudo to be installed. Make it happy.
+      - install -Dm755 sudo /app/bin/sudo
+      - install -Dm755 sudo /app/bin/pkexec
+      # Add entrypoints
+      - install -Dm755 entrypoint.sh /app/bin/entrypoint.sh
+      - install -Dm755 oudedetai /app/bin/oudedetai
+      # Copy metadata information
+      - install -Dm644 appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml
+      - install -Dm644 oudedetai.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 oudedetai.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/oudedetai.png
+      - install -Dm644 logos.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/logos.png
+      - install -Dm644 verbum.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/verbum.png
+      - install -Dm644 oudedetai.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: script
+        dest-filename: sudo
+        commands: []
+      - type: script
+        dest-filename: entrypoint.sh
+        commands:
+          - export WINE_EXE=/app/bin/wine64
+          - export DIALOG=tk
+          - export INSTALLDIR=$XDG_DATA_HOME/Faithlife-Community/oudedetai
+          - /app/bin/oudedetai "$@"
+      - type: file
+        url: https://github.com/FaithLife-Community/LogosLinuxInstaller/releases/download/v4.0.0-beta.8/oudedetai
+        sha256: b07b6d209bdc5271b979ab10b668a433c77f87b63a37918bd5ac519fef9d4aab
+        dest-filename: oudedetai
+      - type: file
+        path: ./icons/oudedetai.png
+        dest-filename: oudedetai.png
+      - type: file
+        path: ./icons/logos.png
+        dest-filename: logos.png
+      - type: file
+        path: ./icons/verbum.png
+        dest-filename: verbum.png
+      - type: file
+        dest-filename: oudedetai.desktop
+        path: ./oudedetai.desktop
+      - type: file
+        dest-filename: appdata.xml
+        path: ./appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --allow=multiarch
+  - --allow=devel
+  - --system-talk-name=org.freedesktop.UDisks2
+  - --system-talk-name=org.freedesktop.NetworkManager
```

Comment:
why are these two needed?

Reply:
I'll try without some of these permissions, it started off with a copy/paste from the wine flatpak (with a couple I know I can remove)